### PR TITLE
Display Producer Statistics

### PR DESCRIPTION
**Pull request by Tthomas63**

### CODE REVIEW

File: src/apps/api/views/producers.py [hunk 1]
```diff
@@ -23,3 +28,22 @@ def create(self, request, *args, **kwargs):
         headers = self.get_success_headers(serializer.data)
 
         return Response({"api_key": serializer.instance.api_key}, status=status.HTTP_201_CREATED, headers=headers)
+
+
+@api_view(['GET'])
+def producer_statistics(request, version):
+    try:
```
Comment (ckcollab) [inline src/apps/api/views/producers.py]: Eeeep! Don't do `try excepts` with no `Exception` to catch, most of the time?

File: src/apps/producers/models.py [hunk 2]
```diff
@@ -9,5 +9,12 @@ class Producer(models.Model):
     contact = models.EmailField(null=True, blank=True)
     url = models.URLField()
 
+    competition_count = models.IntegerField(default=0)
+    dataset_count = models.IntegerField(default=0)
```
Comment (ckcollab) [inline src/apps/producers/models.py]: Missing the migration for these?

**Comment by ckcollab** [comment]
@madclam here's a link to see general stats display, so far:
https://codalabchahub-pr-135.herokuapp.com/

And you can see the screenshots above for comparison, would love a 👍 or 👎

**Comment by ckcollab** [comment]
_NOTE:_ Showing Isabelle this now

**Comment by madclam** [comment]
This looks nice.
- which statistics does this show: for which server? It may be good to show those corresponding to the given query. The remove "chahub brings together".
- when we collapse, this does not result in gaining more width
- what are "datasets"? For the moment we do not push "datasets" to chahub, right? In the future, what will those be (still to be defined).

**Pull request merged**